core-web/libs/block-editor/.storybook/main.js

```diff
@@ -1,11 +1,29 @@
 const rootMain = require('../../../.storybook/main');
 
-// Use the following syntax to add addons!
-// rootMain.addons.push('');
-rootMain.stories.push(
-    ...['../src/lib/**/*.stories.mdx', '../src/lib/**/*.stories.@(js|jsx|ts|tsx)']
-);
 
-module.exports = rootMain;
 
-module.exports.core = { ...module.exports.core, builder: 'webpack5' };
+module.exports = {
+  ...rootMain,
+
+  stories: [
+    ...rootMain.stories,
+    '../src/lib/**/*.stories.mdx',
+    '../src/lib/**/*.stories.@(js|jsx|ts|tsx)'
+  ],
+  addons: [...rootMain.addons
+
+
+  ],
+  webpackFinal: async (config, { configType }) => {
+    // apply any global webpack configs that might have been specified in .storybook/main.js
+    if (rootMain.webpackFinal) {
+      config = await rootMain.webpackFinal(config, { configType });
+    }
+
+
+
+    // add your own webpack tweaks if needed
+
+    return config;
+  },
+};
```

Comment:
I'm guessing this is the new way to add custom config to webpack in Storybook 6.5?